R/nycflights13_sql.R

```diff
@@ -19,8 +19,40 @@
 #' @export
 #' @examples
 #' \dontrun{
-#' # psql_con is a PostgreSQL connnection from .Rprofile
-#' nycflights13_sql(psql_con, schema = "nycflights13")
+#' # we assume credentials is a list specified in .Rprofile
+#'
+#' # odbc + DBI
+#' psql_con <- DBI::dbConnect(
+#'  odbc::odbc(),
+#'  Driver   = "PostgreSQL Unicode",
+#'  Server   = credentials[["host"]],
+#'  Database = "postgres",
+#'  UID      = credentials[["usr"]],
+#'  PWD      = credentials[["pwd"]],
+#'  Port     = 5432
+#' )
+#'
+#' # RPostgreSQL + DBI
+#' psql_con <- RPostgreSQL::dbConnect(
+#'  drv = DBI::dbDriver("PostgreSQL"),
+#'  host = credentials[["host"]],
+#'  dbname = "postgres",
+#'  user = credentials[["usr"]],
+#'  password = credentials[["pwd"]],
+#'  port     = 5432
+#' )
+#'
+#' # RPostgres + DBI
+#' psql_con <- DBI::dbConnect(
+#'  drv = RPostgres::Postgres(),
+#'  host = credentials[["host"]],
+#'  dbname = "postgres",
+#'  user = credentials[["usr"]],
+#'  password = credentials[["pwd"]],
+#'  port = 5432
```

Comment:
The only thing I would change here is maybe use our fake/Travis credentials so that we don’t have to explain the format of the, in the .rprofile etc. hopefully most people looking at the examples will know to sub in their own username/password etc